igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/TravisBuildMonitor.groovy

```diff
@@ -164,11 +187,14 @@ class TravisBuildMonitor extends CommonPollingMonitor {
                         buildCache.setLastBuild(master, job, oldBuildNumber, oldBuildBuilding, buildCacheJobTTLSeconds())
                     }
                 }
-
             }
         }
     }
 
+    private static String prettyPrintBuild(V3Build build) {
+        return "[" + build.repository.slug + ":" + build.number + ":" + build.state + "]"
+    }
```

Comment:
Not sure if its doable or better, but you could consider moving this to `V3Build`s `toString()` method?

Reply:
Good idea